cdptools/bin/clone_db.py

```diff
@@ -25,9 +25,13 @@ def __parse(self):
             prog="clone_prod",
             description="Clone prod database to target database."
         )
-        p.add_argument("project_id", help="Project id of source database.", default="cdp-seattle")
+
         p.add_argument("target_credentials", type=Path, help="Credentials of target database.")
         p.add_argument("--debug", action="store_true", dest="debug", help="Show traceback if the script were to fail.")
+        group = p.add_mutually_exclusive_group(required=True)
+        group.add_argument('--source_project_id', help="Project id of source database.", default="cdp-seattle")
```

Comment:
Can you remove the default value here. IMO defaults shouldn't be allowed for a clone because user should be _very_ explicit.